packages/forms/src/UIForm/Widget/condition.test.js

```diff
@@ -49,6 +50,11 @@ const TRUTHY_CONDITIONS = [
 		strategy: 'length',
 		path: 'arrayEmpty',
 	},
+	{
+		values: ['0'],
+		strategy: 'length',
+		path: 'arrayEmpty',
+	},
```

Comment:
no difference with the previous case ?

Reply:
it is to check the coercing of the value (string/number)

Reply:
👍